services/git-operator/0.1.0/git-operator-manifests/all.yaml

```diff
@@ -741,6 +934,205 @@ spec:
       serviceAccountName: git-operator-controller-manager
       terminationGracePeriodSeconds: 10
 ---
+apiVersion: apps/v1
```

Comment:
i am curious - how were we creating Statefulset so far (like in dev.11) ?

Reply:
They were embedded in the operator binary